go/vt/vtgate/plugin_mysql_server_test.go

```diff
@@ -348,7 +348,7 @@ func TestGracefulShutdown(t *testing.T) {
 
 	vh := newVtgateHandler(&VTGate{executor: executor, timings: timings, rowsReturned: rowsReturned, rowsAffected: rowsAffected})
 	th := &testHandler{}
-	listener, err := mysql.NewListener("tcp", "127.0.0.1:", mysql.NewAuthServerNone(), th, 0, 0, false, false, 0)
+	listener, err := mysql.NewListener("tcp", "127.0.0.1:", mysql.NewAuthServerNone(), th, 0, 0, false, false, 0, 0, "8.0.30-Vitess")
```

Comment:
It's not an exposed constant, so it can't be used here. I can turn these two into their own constant? I thought about it and didn't know if it was worth it for 2 or not then.

Reply:
yeah tests-only seems fine this way